Formula/vitess.rb

```diff
@@ -0,0 +1,41 @@
+class Vitess < Formula
+  desc "Database clustering system for horizontal scaling of MySQL"
+  homepage "https://vitess.io"
+  url "https://github.com/vitessio/vitess/archive/v8.0.0.tar.gz"
+  sha256 "c47320b9bcb874b1a6dfca78ec677be7c4bb4c7b2a6470df80bd1bc0ad125e92"
+  license "Apache-2.0"
+
+  depends_on "automake" => :build
+  depends_on "go" => :build
+  depends_on "etcd"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}", "VTROOT=#{buildpath}"
+    bin.install "bin/mysqlctl"
```

Comment:
Are `mysqlctl` and `vtctl` used outside of our tests? If not, they should go into `pkgshare` as well

Reply:
These aren't used in Homebrew tests, but are needed if users want to follow the Getting Started tutorial on the Vitess website.